server/src/main/java/io/crate/execution/ddl/tables/TransportCloseTable.java

```diff
@@ -74,24 +74,21 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
-import javax.annotation.Nullable;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.EnumSet;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.function.Consumer;
-import java.util.stream.Collectors;
 
 public final class TransportCloseTable extends TransportMasterNodeAction<CloseTableRequest, AcknowledgedResponse> {
 
     private static final Logger LOGGER = LogManager.getLogger(TransportCloseTable.class);
```

Comment:
same as for `TransportOpenCloseTableOrPartitionAction`:

 reverted to https://github.com/crate/crate/pull/12231 - first change before https://github.com/crate/crate/commit/6857a28b669b6e6f056e59cbfdc760c59739701c and https://github.com/crate/crate/commit/562abc8a358c344e3986cc1f1a27e06cec65f43c + kept changes from https://github.com/crate/crate/pull/12370 and https://github.com/crate/crate/pull/12378